### PR TITLE
backmerge latest

### PR DESCRIPTION
**Pull request by technophile-04**
### Description: 

- reverse rpc fallback order (https://github.com/scaffold-eth/scaffold-eth-2/pull/1010)
- Encrypt deployer PK on .env file (when using hardhat) (https://github.com/scaffold-eth/scaffold-eth-2/pull/1008 )

### CODE REVIEW

**Comment by technophile-04** [comment]
Will merge #166 after we merge this

**Comment by technophile-04** [comment]
Just pushed a small changes https://github.com/scaffold-eth/create-eth/pull/169/commits/a88dd0424c739a4721b60e600f472ebddfeeab2d

actually, if you do `npx create-eth@latest -e ponder -s foundry` (basically when using extension) it was not adding default deployment.

**Review by rin-st — APPROVED** [review]
Lgtm!

**Pull request merged**
